### ISSUE TITLE
Introduce variable for OpenSSL binary

### ISSUE THREAD

**Issue by warrenski** [Neilpang/acme.sh#320]
Please add support for an optional variable in `account.conf` whereby a user can define the path to the OpenSSL binary.

For example, on systems with an older OpenSSL package, the sysadmin may have compiled a separate standalone version of modern OpenSSL. The variable would allow the user to override the system default. For example:

`OPENSSL_BIN="/opt/openssl/bin/openssl"`

**Comment by Neilpang** [comment]
Good idea, I will fix it soon.
Here is a temp workaround for you:
```
export PATH=/opt/openssl/bin/:$PATH
acme.sh --issue .......
```

**Comment by Neilpang** [comment]
fixed  at  a746139c53b26052b6eec79c16fd100204d4e04c
